pnl-subgraph/src/ExchangeMapping.ts

```diff
@@ -0,0 +1,41 @@
+import { OrderFilled } from './types/Exchange/Exchange';
+
+import { parseOrderFilled } from './utils/parseOrderFilled';
+import { updateUserPositionWithBuy } from './utils/updateUserPositionWithBuy';
+import { updateUserPositionWithSell } from './utils/updateUserPositionWithSell';
+
+import { COLLATERAL_SCALE } from '../../common/constants';
+
+/**
+ * Handles individual OrderFilled events
+event OrderFilled(
+    bytes32 indexed orderHash,
+    address indexed maker,
+    address indexed taker,
+    uint256 makerAssetId,
+    uint256 takerAssetId,
+    uint256 makerAmountFilled,
+    uint256 takerAmountFilled,
+    uint256 fee
+);
+ * @param event
+ */
+export function handleOrderFilled(event: OrderFilled): void {
```

Comment:
either the buyer or the seller will be the exchange contract correct? should we ignore that side?

Reply:
yes! good catch ty